flocker/provision/_install.py

```diff
@@ -141,17 +141,38 @@ def task_enable_docker():
     ]
 
 
-def task_disable_firewall():
+def configure_firewalld(rule):
```

Comment:
If we're relying on firewall rules in the package, why do we need to do it in the provisioning system?